packages/airnode-adapter/e2e/index.ts

```diff
@@ -294,17 +291,16 @@ describe('Extraction, encoding and simple on chain decoding', () => {
     });
 
     it('throws on int256 max number', async () => {
-      try {
-        await testDecoder['decodeSignedInt256'](
+      const goExtractAndEncode = await go(() =>
+        testDecoder['decodeSignedInt256'](
           extractAndEncode({
             _type: 'int256',
             _path: 'big.tooLargeDecimal',
           })
-        );
-        expect.fail();
-      } catch (e: any) {
-        expect(e.message).to.contain('out-of-bounds');
-      }
+        )
+      );
+      expect(goExtractAndEncode.success).to.equal(false);
```

Comment:
You can use `assertGoError` instead of this assertion and the typecast below. (same for others).

Reply:
This is cool, thanks for pointing it out!